go-chaos/cmd/cluster.go

```diff
@@ -45,11 +46,88 @@ func AddClusterCommands(rootCmd *cobra.Command, flags *Flags) {
 			return waitForChange(flags)
 		},
 	}
+	var scaleCommand = &cobra.Command{
+		Use:   "scale",
+		Short: "Scales the cluster to the given size",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return scaleCluster(flags)
+		},
+	}
 
 	rootCmd.AddCommand(clusterCommand)
 	clusterCommand.AddCommand(statusCommand)
 	clusterCommand.AddCommand(waitCommand)
 	waitCommand.Flags().IntVar(&flags.changeId, "changeId", 0, "The id of the change to wait for")
+	clusterCommand.AddCommand(scaleCommand)
+	scaleCommand.Flags().IntVar(&flags.brokers, "brokers", 0, "The amount of brokers to scale to")
+}
+
+func scaleCluster(flags *Flags) error {
+	k8Client, err := createK8ClientWithFlags(flags)
+	if err != nil {
+		panic(err)
+	}
+
+	err = k8Client.AwaitReadiness()
+	if err != nil {
+		return err
+	}
+
+	port := 9600
+	closePortForward := k8Client.MustGatewayPortForward(port, port)
+	defer closePortForward()
+
+	changeResponse, err := requestBrokerScaling(port, flags.brokers)
+	if err != nil {
+		return err
+	}
+	formatted, err := json.MarshalIndent(changeResponse, "", "  ")
+	if err != nil {
+		return err
+	}
+	fmt.Println(string(formatted))
```

Comment:
❌ Sorry I have overseen this before. Can you change this and also on the other subcommand to use the `internal.LogInfo` or `internal.LogVerbose` depending on the verbose flag you can then control whether it is logged or not. Furthermore, it adds additional context with json logging enabled.